qemu/tests/qemu_guest_agent.py

```diff
@@ -343,6 +343,23 @@ def gagent_verify(self, params, vm):
         self.gagent.verify_responsive()
         logging.info(self.gagent.cmd("guest-info"))
 
+    @error_context.context_aware
+    def gagent_setsebool_verify(self, params, vm):
```

Comment:
if it's better to use function name of  gagent_setsebool_on()?
And can you add docstring for this function?